packages/ckeditor5-image/tests/image/imageediting.js

```diff
@@ -295,6 +430,17 @@ describe( 'ImageEditing', () => {
 					.to.equal(
 						'<image alt="alt text" src="/assets/sample.png" srcset="{"data":"small.png 148w, big.png 1024w"}"></image>'
 					);
+
+				editor.setData(
+					'<p><img src="/assets/sample.png" alt="alt text" srcset="small.png 148w, big.png 1024w" /></p>'
+				);
+
+				/* eslint-disable max-len */
```

Comment:
Maybe it would be better to split the lines below instead? I can see there are multiple places where you disable this liner rule.